docs/index.md

```diff
@@ -3,11 +3,13 @@
 page_title: "Coder Provider"
 subcategory: "Infrastructure"
 description: |-
-  
+    Terraform provider for managing Coder templates, which are the underlying infrastructure for Coder workspaces.
```

Comment:
Managing templates still may be a bit confusing as the coderd provider does too. Maybe we say something about "workspace internals"

Reply:
What if we call it `defining templates`?
```suggestion
    Terraform provider for defining/building Coder templates, which are the underlying infrastructure for Coder workspaces.
```

Reply:
Let's backport this change first and iterate on the exact wording later.